tests/flow/test_concurrent_query.py

```diff
@@ -12,45 +12,27 @@ def query_aggregate(query):
     env = Env(decodeResponses=True)
     conn = env.getConnection()
     graph = Graph(GRAPH_ID, conn)
-
-    actual_result = graph.query(query)
-    person_count = actual_result.result_set[0][0]
-    if person_count != len(people):
-        return False
-    
-    return True
+    return graph.query(query)
 
 def query_neighbors(query):
```

Comment:
What's the difference between:
`query_aggregate `, `query_neighbors `, `query_write ` and `thread_run_query`
seems just need `thread_run_query`